packages/ui-library/src/elements/autocomplete/style.css

```diff
@@ -40,6 +40,7 @@
 			focus-within:yst-border-primary-500
 			focus-within:yst-outline-none
 			focus-within:yst-ring-1
+			yst-shadow-sm
```

Comment:
Not sure why the shadow class is added in the middle of the focus ones?

Reply:
Good point! I moved it to be under the `yst-rounded-sm`, same as in the 'input-text` style.